Library/Formula/fdm.rb

```diff
@@ -0,0 +1,21 @@
+class Fdm < Formula
+  homepage "http://sourceforge.net/projects/fdm/"
+  url "https://downloads.sourceforge.net/project/fdm/fdm/fdm-1.8/fdm-1.8.tar.gz"
+  sha1 "25c0ebc0c2b43984d04c82d0ba3909acf7403df7"
+
+  depends_on "tdb"
```

Comment:
@mikemcquaid Hmm... it seems like tdb was originally created as part of the samba project... it's possible that samba has a more up to date implementation.  I'll dig around a bit more and see what I can find.